dataprofiler/tests/profilers/test_utils.py

```diff
@@ -61,7 +61,7 @@ def test_find_diff(self):
 
         # Ensure lists and sets are handled appropriately
         self.assertEqual(
-            "unchanged", utils.find_diff_of_lists_and_sets([3, 2], [2, 3, 2])
+            [[], [3, 2], [2]], utils.find_diff_of_lists_and_sets([3, 2], [2, 3, 2])
```

Comment:
updated outputs based on the changes made to `find_diff_of_lists_and_sets(stat1, stat2)`